.codecov.yml

```diff
@@ -4,3 +4,6 @@ coverage:
     project:
       default:
         threshold: 2
+        tests:
```

Comment:
I think this should be `javascript` and one indentation to the left..

Reply:
Based on my understanding of this:
https://docs.codecov.io/docs/commit-status#section-excluding-tests-example